lib/streamlit/elements/image.py

```diff
@@ -207,6 +207,8 @@ def _normalize_to_bytes(data, width, output_format):
     if output_format.lower() == "auto":
         ext = imghdr.what(None, data)
         mimetype = mimetypes.guess_type("image.%s" % ext)[0]
+        if mimetype is None:
```

Comment:
I realize I wrote this, but I would just add a comment that suggests that there's no other options, just try to convert it (would require Pillow support).